scala/bench/src/main/scala/svc/EqualBench.scala

```diff
@@ -18,27 +18,31 @@ class EqualBench {
   var list1: List[Int] = _
   var listK1: List[Kitties.Foo] = _
   var listK2: List[Kitties.Foo] = _
-  var listZ1: List[Zed.Foo] = _
-  var listZ2: List[Zed.Foo] = _
+  var listz0: List[Int] = _
+  var listz1: List[Int] = _
+  var listZ1: scalaz.IList[Zed.Foo] = _
+  var listZ2: scalaz.IList[Zed.Foo] = _
```

Comment:
This grabs the `IList` local to this PR and not the one from the scalaz dep? They both have the same package and type name, so I'm confused how that works.

Reply:
Monkey patch. Upstream will use this soon.

Reply:
Well, it's broken so probably not :-D I'll fix it.